_articles/system-firmware.md

```diff
@@ -18,52 +18,93 @@ These instructions are for System76 owners who have been prompted for a firmware
-2. Select **Restart Now** from the notification or **Update** from the Settings application.
-3. The System76 Firmware updater will start on reboot. Once it has initialized, you'll be prompted to **press <kbd>Enter</kbd>** to commence flashing.
-4. The firmware update process will begin, then the system will restart back into the System76 Firmware updater. The fans on the system will be on full blast.
-5. The firmware update process will finish. When prompted, **press any key to shutdown**.
-6. After the system powers off, **press the power button** to turn it back on.
-7. Two small blue boxes may appear in succession with a message about the CMOS. **Press <kbd>Enter</kbd>** to dismiss each one.
-8. The machine will then reboot. Repeatedly press or hold the <kbd>F2</kbd> key to boot into the BIOS settings. Using the arrow keys, go over to the **Exit** tab.
- - If the BIOS settings do not launch, reboot the machine and try again.
-9. Once in the BIOS move to the Exit tab and select then confirm **Load Setup Defaults** or **Load Optimal Defaults**. (This step does not apply to Open Firmware machines)
-10. Select and confirm **Save changes and reset** in that same tab.
-11. Once the system restarts, the firmware will be up to date!
-
-The following commands can be used to schedule the firmware update for the next reboot and then reboot the system:
+![Starting a firmware update (Ubuntu)](/images/system-firmware/install-ubuntu.jpg)
+
+### Starting the Update (CLI)
+
+The command-line firmware utility can be installed using this command:
+
+```
+sudo apt install system76-firmware
+```
+
+Once installed, the following command can be used to schedule a firmware update for the next reboot (if the firmware is already up-to-date, this command will schedule a re-installation of the current firmware):
 
 ```
 sudo system76-firmware-cli schedule
-sudo reboot
 ```
 
+Reboot the system using `sudo systemctl reboot` to proceed with the update, or use `sudo system76-firmware-cli unschedule` to cancel the update.
+
+### Performing the Update
+
+The System76 Firmware Updater will start on the next boot. Once it has initialized, you'll be prompted to **press <kbd>Enter</kbd>** to commence flashing.
+
+![Press Enter to commence flashing](/images/system-firmware/press-enter.jpg)
+
+Depending on the model, the system may reboot at this point. The system's fans may spin at full speed while the new firmware is being written. Once the firmware update process is complete, you may see a prompt to **press any key** to shut down.
+
+![Press any key to shut down](/images/system-firmware/press-any-key.jpg)
+
+After the system powers off, **press the power button** to turn it back on. On machines running Open Firmware, the system should boot normally.
+
+### Disabling the ME
+
+For machines not running Open Firmware, a few extra steps may be required after an update to ensure the Intel ME is disabled. 
```

Comment:
Should we change this to, "For laptops not running Open Firmware, ..."? People will probably reference this for desktops which don't have disable-able ME.

Reply:
For some reason, I thought "may" would alert people that section might not apply (along with the images), but you're right, it's better to specify. Changed in https://github.com/system76/docs/pull/426/commits/f6b3dc59147503131a62c808f052e17ba91c1dee.